.github/workflows/update_version.yml

```diff
@@ -1,11 +1,10 @@
 name: Update package version
 on:
   create:
-    branches:
-      - 'release/**'
 jobs:
   UPDATE:
     runs-on: ubuntu-latest
+    if: startsWith('ref/heads/release/', github.ref)
```

Comment:
Please remove the if statement and use the right events in the on part.
https://help.github.com/en/actions/reference/events-that-trigger-workflows

Reply:
Okay than the way you select is the best way.